inc/umock_c/umock_c_internal.h

```diff
@@ -931,9 +931,9 @@ typedef struct MOCK_CALL_METADATA_TAG
 #define UNUSED_ARG(arg_type, arg_name) \
     (void)arg_name;
 
-/* Codes_SRS_UMOCK_C_LIB_01_205: [ If IGNORED_PTR_ARG or IGNORED_NUM_ARG is used as an argument value with STRICT_EXPECTED_CALL, the argument shall be automatically ignored. ]*/
-/* Codes_SRS_UMOCK_C_LIB_01_206: [ IGNORED_PTR_ARG shall be defined as NULL so that it can be used for pointer type arguments. ]*/
-/* Codes_SRS_UMOCK_C_LIB_01_207: [ IGNORED_NUM_ARG shall be defined to 0 so that it can be used for numeric type arguments. ]*/
+/* Codes_SRS_UMOCK_C_LIB_01_205: [ If IGNORED_ARG or IGNORED_ARG is used as an argument value with STRICT_EXPECTED_CALL, the argument shall be automatically ignored. ]*/
```

Comment:
>IGNORED_ARG or IGNORED_ARG [](start = 38, length = 26)

If "popescu" or "popescu".

Remove one Popescu

Reply:
One Popescu died.

---
In reply to: [395824099](https://github.com/Azure/umock-c/pull/152#discussion_r395824099) [](ancestors = 395824099)